app/models/episode.rb

```diff
@@ -214,11 +217,19 @@ def explicit_content
   end
 
   def item_guid
-    original_guid || self.class.generate_item_guid(podcast_id, guid)
+    original_guid || generate_item_guid
   end
 
   def item_guid=(new_guid)
-    self.original_guid = new_guid.blank? ? nil : new_guid
+    self.original_guid = (new_guid.blank? || new_guid == generate_item_guid) ? nil : new_guid
```

Comment:
Unrelated, but I realized that the UI was saving that generated `prx_123_abcd` value back to the database.  Changing the column from nil to non-nil.

Wasn't really hurting anything... but to be consistent, we should keep these nil.